src/app/components/product/productItem/ProductItem.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render } from 'tests';
+import { within } from '@testing-library/react';
```

Comment:
Z tego co widziałem, to eksportujesz wszystko z `test/index.ts`, więc `within` można wyciągnąć razem w linii pierwszej, prawda?